sklearn/cluster/k_means_.py

```diff
@@ -792,6 +792,21 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
     inertia_ : float
         Sum of distances of samples to their closest cluster center.
 
+    Examples
+    --------
+
+    >>> from sklearn.cluster import KMeans
+    >>> import numpy as np
+    >>> X = np.array([[1, 2], [0, -4], [9, 1], [0, 0]])
+    >>> kmeans = KMeans(n_clusters=4, random_state=0).fit(X)
+    >>> kmeans.predict([[0, 0], [1, 2]])
+    array([3, 0], dtype=int32)
+    >>> kmeans.transform(X)
```

Comment:
I think it would be nice to show the cluster labels (`kmeans.labels_`) rather than the `transform`...